Mathlib/Geometry/Synthetic/Avigad/Axioms.lean

```diff
@@ -0,0 +1,251 @@
+/-
+Copyright (c) 2023 André Hernandez-Espiet. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: André Hernandez-Espiet
+-/
+import Mathlib.Data.Real.Basic
+import Mathlib.Data.Set.Finite
+
+/-!
+Axioms of synthetic geometry
+-/
+
+/-! Universes for points lines and circles-/
+universe u
+
+/-- `incidence_geometry` represents geometry in the Euclidean sense, with primitives for points
+lines and circles-/
+class incidence_geometry :=
+/--Points in the plane-/
+(point : Type u)
+/--Lines in the plane-/
+(line : Type u)
+/--Circles in the plane-/
+(circle : Type u)
+
+/--A point being on a line-/
+(online : point → line → Prop)
+/--Two points being on the sameside of a line-/
+(sameside : point → point → line → Prop)
+/--Three points being in a row-/
+(B : point → point → point → Prop)
+/--A point being the center of a center-/
+(center_circle : point → circle → Prop)
+/--A point being on a circle-/
+(on_circle : point → circle → Prop)
+/--A point being inside a circle-/
+(in_circle : point → circle → Prop)
+/--That two lines intersect-/
+(lines_inter : line → line → Prop)
+/--A line and a circle intersect-/
+(line_circle_inter : line → circle → Prop)
+/--Two circles intersect-/
+(circles_inter : circle → circle → Prop)
```

Comment:
The mathlib 4 [naming guide](https://leanprover-community.github.io/contribute/naming.html) says these `Prop` and `Type` fields should be named as `Point`, `OnCircle`, etc, with no underscores and PascalCase.

What you have looks like Lean 3 naming conventions!

Reply:
Ok, I think I have changed what you requested. What's the next thing you would like me to revise?

Reply:
Another lean4ism is using `where` instead of `:=` and not using parentheses around the fields. Not sure if it's enforced style.

Reply:
Ok, I made the parenthesis and `:=` suggestions. What else would you like me to modify?